dashboard/test/lib/queue_processor_test.rb

```diff
@@ -0,0 +1,187 @@
+# Tests for the SQS::QueueProcessor class. These are run by default against a fake
+# SQS instance but can be run against the real SQS by setting the environment variable
+# USE_REAL_SQS to 'true'.  If LOG_TO_STDOUT is true, logs to stdout, otherwise uses the
+# Rails logger.
```

Comment:
If this `LOG_TO_STDOUT` switch was useful for test-driven development in this case, we might want to make our existing test-environment loggers more configurable via an environment switch like this in the future.